### PR TITLE
London 9 - Ying Xing - SQL - Full-Stack-Project-Assessment

### PR DESCRIPTION
**Pull request by LexiisYing**

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+/* .hero {
+  background-color: #f9f9f9;
+  padding: 20px;
+  text-align: center;
+}
+
+.hero-title {
+  font-size: 28px;
+  margin-bottom: 20px;
+} */
```
Comment (sampennington) [inline client/src/App.css:32]: Try and delete code you dont need anymore, it gets messy with it all left around 😄

File: client/src/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  useEffect(() => {
+    fetch("https://musicvideos-app.onrender.com")
+      .then((response) => response.json())
+      .then((data) => {
+        // Order the videos by rating in descending order
+        //const sortedVideos = data.sort((a, b) => b.rating - a.rating);
+        setVideos(data);
+      })
+      .catch((error) => {
+        console.error("Error fetching data:", error);
```
Comment (sampennington) [inline client/src/HomePage.js:43]: Error handling 🙌

File: server/server.js [hunk 3]
```diff
@@ -1,15 +1,201 @@
...
+    if (!video) {
+      return res
+        .status(404)
+        .json({ result: "failure", message: "Video not found." });
+    }
+
+    res.json(video);
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({ error: "Internal server error" });
```
Comment (sampennington) [inline server/server.js:94]: Considering various types of erorr, great!

**Review by sampennington — COMMENTED** [review]
Generally looks pretty good, well done!!
Just rememebr to clear up your code a bit, it make it hard to review. Theres a bunch of comments and code not being used anymore

File: client/src/App.css (hunk 1 above)
Comment (LexiisYing) [reply client/src/App.css:32]: just thought i'd need them in the future lol

**Pull request closed without merging**